presto-docs/src/main/sphinx/release/release-0.242.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.242
+=============
+
+General Changes
+_______________
+* Fix compiler error in LambdaBytecodeGenerator when CSE is enabled and multiple SQL functions contain lambda expressions.
+* Fix possible permanent stuck queries when memory limit exceeded during shuffle.
+* Fix a bug where ``IS DISTINCT FROM NULL`` produces wrong results for columns with null values.
+* Add config ``experimental.spiller.task-spilling-strategy`` for choosing different spilling strategy to use.
+* Add session property ``query_max_output_bytes`` and configuration property ``query.max-output-bytes`` to control how much data a query can output.
+* Add support for file renaming for Hive connector. This can be enabled with ``hive.file_renaming_enabled`` configuration property.
+* Add support for fragment result caching. When enabled, if the same plan fragment and same connector split hit the same worker, engine would directly fetch result from cache and skip computation. Currently only partial aggregation is supported. Cache could be enabled by setting ``fragment-result-cache.enabled`` to ``true`` and tuned by other configs started with ``fragment-result-cache``. Query could use fragment result cache by setting config ``experimental.fragment-result-caching-enabled`` or session property ``fragment_result_caching_enabled`` to ``true``.
+* Add support to query parquet files written by parquet-avro library that uses TIMESTAMP_MICROS as the OriginalType to represent timestamp.
+* Add support for partition versioning. This can be enabled with ``hive.partition-versioning-enabled`` configuration property
+* Add support for caching HiveMetastore calls at a more granular level . Supported levels:  'PARTITION' and 'ALL'(default). This can be set with ``hive.metastore-cache-scope`` configuration property.
+
+Hive Changes
+_______________
+* Fix a bug where DWRF encryption would fail for large uncompressed column values.
+* Fix a bug where non-Presto readers could not read encrypted DWRF files written by Presto if the encryption group listed columns out of order.
+* Fix a performance regression for String field handling in GenericHiveRecordCursor when the SerDe does not provide an efficient Writable implementation.
+* Improve split loading thread utilization by reducing the number of tasks submitted for small scans.
+
+Pinot Changes
+_______________
+* Adding config: ``pinot.pushdown-topn-broker-queries`` to allow pushing down TOPN queries to Pinot.
+
+Cassandra Change
+_______________
+* Add ``SMALLINT``, ``TINYINT``, and ``DATE`` type support to Cassandra connector.
+
+Druid Changes
+_______________
+* Upgrade apache druid version to 0.19.0
```

Comment:
@zhenxiao What is the user-facing impact here?

Reply:
no user facing impact. shall we mention it?

Reply:
@zhenxiao No, we should only include the ones with user-facing impact. See also [Release Notes Guideline](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines)

Reply:
let's remove it